include/ipc.hpp

```diff
@@ -26,16 +26,9 @@ static constexpr auto kDefaultSocketTimeout = std::chrono::seconds{2};
 class UnixSocket {
 public:
   using socket_t = int;
-  UnixSocket() noexcept = default;
 
   explicit UnixSocket(socket_t handle) noexcept : _handle(handle) {}
 
-  UnixSocket(UnixSocket &&socket) noexcept = default;
-  UnixSocket(const UnixSocket &) = delete;
-
-  UnixSocket &operator=(const UnixSocket &) = delete;
```

Comment:
Same question here: these had a purpose ?

Reply:
Same as above, no destructor -> no need to delete copy constructors.
Here UniqueFd member already prevents copy.